docs/source/common/loggers.rst

```diff
@@ -226,8 +231,8 @@ The :class:`~pytorch_lightning.loggers.WandbLogger` is available anywhere except
     class MyModule(LightningModule):
         def any_lightning_module_function_or_hook(self):
             some_img = fake_image()
-            self.logger.experiment.log({
-                 "generated_images": [wandb.Image(some_img, caption="...")]
+            self.log({
+                "generated_images": [wandb.Image(some_img, caption="...")]
             })
```

Comment:
It actually does! I do it [here](https://colab.research.google.com/drive/1Tr9dYlwBKk6-LgLKGO8KYZULnguVA992?usp=sharing#scrollTo=2diE1gzlzpqZ).
Note : you may want to about the learning rate finder section due to current issue https://github.com/PyTorchLightning/pytorch-lightning/issues/6471

It presents some advantage when reloading the Trainer later as this logging becomes a noop until adding a logger (which would raise an issue calling ’self.logger.experiment...’).

The issue may be when several loggers are present, it will raise an error with the other ones.

Let me know if it's not supposed to be used this way and I'll update the doc and upcoming demo.

Reply:
ok, I didn't know that we support this